src/main/kotlin/org/openmbee/flexo/mms/routes/sparql/RepoQuery.kt

```diff
@@ -0,0 +1,25 @@
+package org.openmbee.flexo.mms.routes.sparql
+
+import io.ktor.server.routing.*
+import org.openmbee.flexo.mms.REPO_QUERY_CONDITIONS
+import org.openmbee.flexo.mms.assertPreconditions
+import org.openmbee.flexo.mms.server.sparqlQuery
+import org.openmbee.flexo.mms.processAndSubmitUserQuery
+
+
+/**
+ * User submitted SPARQL Query to a specific repo
+ */
+fun Route.queryRepo() {
+    sparqlQuery("/orgs/{orgId}/repos/{repoId}/query/{inspect?}") {
+        parsePathParams {
+            org()
+            repo()
+            inspect()
+        }
+
+        processAndSubmitUserQuery(requestContext, prefixes["mor"]!!, REPO_QUERY_CONDITIONS.append {
+            assertPreconditions(this)
+        }, false, prefixes["mor"])
```

Comment:
passing in prefixes["mor"] causes the added FROM and FROM NAMED graphs to be malformed, can remove the last 2 params here

Reply:
can you elaborate? the last argument sets the base IRI for the query, which determines how relative IRIs resolve, i.e., `<foo>` would be equivalent to `mor:foo`

Reply:
i guess the serialize isn't adding the base to the output string, but if it does, would it affect any prefixes in the user query? for repo query `PREFIX  mms:  <https://mms.openmbee.org/rdf/ontology/>` would be a common prefix.
Also even if it adds the base, the graph iri is still wrong, shouldn't it be <graphs/Metadata> if taking into account base?